src/main/java/controller/Controller.java

```diff
@@ -0,0 +1,61 @@
+package controller;
+
+import domain.Lotto;
+import domain.Lottos;
+import domain.PrizeResult;
+import domain.WinningLotto;
+import java.util.ArrayList;
+import java.util.List;
+import service.Judgement;
+import service.LottoMaker;
+import service.parser.BonusNumberParser;
+import service.parser.MoneyParser;
+import service.parser.WinningNumberParser;
+import view.InputView;
+import view.OutputView;
+
+public class Controller {
+    private final InputView inputView;
+    private final OutputView outputView;
+    private final LottoMaker lottoMaker;
+
+    public Controller(InputView inputView, OutputView outputView, LottoMaker lottoMaker) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+        this.lottoMaker = lottoMaker;
+    }
+
+    public void start() {
+        int lottoCount = inputLottoMoney();
+
+        Lottos lottos = buyLotto(lottoCount);
+        outputView.displayLottoNumbers(lottos);
+
+        List<Integer> winningNumbers = inputWinningNumber();
+        int bonusNumber = inputBonusNumber(winningNumbers);
+
+        PrizeResult prizeResult = Judgement.judge(lottos, new WinningLotto(winningNumbers, bonusNumber));
+        outputView.displayPrizeSummary(prizeResult);
+    }
+
+    private Lottos buyLotto(int count) {
+        ArrayList<Lotto> lottos = new ArrayList<>();
```

Comment:
```suggestion
        List<Lotto> lottos = new ArrayList<>();
```
로 개선할 수 있을 것 같아요. 어떤 차이가 있을까요?

Reply:
캐치 못했던 부분이네요! 가장 큰 차이점은 List는 인터페이스이고 ArrayList는 구현체입니다. 그리고 두 개의 리스트 중 어느 리스트에 의존하느냐에 따라 코드의 수정 범위가 달라집니다. 

ArrayList라는 구현체에 의존하게 되면 만약 Lotto를 모아둔 리스트 즉, 구매한 복권들에 대한 리스트가 ArrayList에서 LinkedList로 변경해야 되는 상황에서 의존 관계도 함께 변경해야 하는 경우가 생길 수 있습니다. 하지만, List라는 인터페이스에 의존하게 되면 LinkedList로 객체를 변경해주기만 하면 됩니다!

Reply:
좋습니다~
이펙티브 자바 64장. [객체는 인터페이스를 사용해 참조하라](https://agentsmith.tistory.com/48) 참고해주세요 :)